templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -388,8 +388,8 @@ Parameters:
     AllowedPattern: >-
       ^(?=^.{8,255}$)(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[^A-Za-z0-9])(?!.*[@/"']).*$
     ConstraintDescription: >-
-      Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol
-    Description: Password for the master ('postgres') account. Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol
+      Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, and 1 of the following symbols: ! # $ { * : [ = , ] - _ @ + % &
```

Comment:
Do you know if there was any issue with the `@` symbol? I wonder why it was excluded previously.

Reply:
So based on the taskcat `template_params.py` file `@` should be supported as defined by the variable: `specialchars = "!#$&{*:[=,]-_%@+"`. No sure why it was missed previously.